### PR TITLE
feat: create eviva queue resources

### PR DESCRIPTION
**Pull request by petter-a**
## Purpose
When sending an application to Viva, the request sometimes fails due to instability. To work around this shortcoming and avoid manual labour to correct applications, each application is added to an SQS queue when the state has reached PDF_GENERATED. The queue has a lambda trigger that executes submitApplication on regular basis until the application has been successfully submitted.

install by issuing:

```
# checkout PR to GitHub library
cd services/queues/viva
sls deploy
```

### CODE REVIEW

File: services/queues/viva/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                status:
+                  M:
+                    type:
+                      S: [{ "prefix": "active:submitted" }]
+                state:
+                  S: ["PDF_GENERATED"]
+
+  Outputs:
+    VivaSubmissionQueueArn:
+      Value:
```
Comment (karlknowit) [inline services/queues/viva/serverless.yml:55]:
```yaml
Value: !GetAtt VivaSubmissionQueue.Arn
```

File: services/queues/viva/serverless.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        Fn::GetAtt:
+          - VivaSubmissionQueue
+          - Arn
+      Export:
+        Name: ${self:custom.stage}-VivaSubmissionQueue
```
Comment (karlknowit) [inline services/queues/viva/serverless.yml:60]: You are exportin the Arn of the queue, perhaps include `Arn` in the export name?

```yaml
Name: ${self:custom.stage}-VivaSubmissionQueueArn
```

**Review by karlknowit — APPROVED** [review]

File: services/queues/viva/serverless.yml (hunk 1 above)
Comment (petter-a) [reply services/queues/viva/serverless.yml]: fixed

File: services/queues/viva/serverless.yml (hunk 2 above)
Comment (petter-a) [reply services/queues/viva/serverless.yml]: Fixed

**Pull request merged**